internal/gengapic/testdata/empty_client_init.want

```diff
@@ -41,37 +58,40 @@ func NewClient(ctx context.Context, opts ...option.ClientOption) (*Client, error
 	if err != nil {
 		return nil, err
 	}
-	c := &Client{
+	client := Client{CallOptions: defaultCallOptions()}
```

Comment:
Seems like we could move this down right before the `return` to avoid mixing struct initialization here with assignment below.

Reply:
We need to leave it where it is so that the embedded client can get a pointer to the call options and the LROClient, if one exists.

Reply:
Ah, gotcha. Sounds good.